railties/lib/rails/generators/rails/app/templates/app/views/layouts/application.html.erb.tt

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang=<%%= I18n.locale %>>
```

Comment:
Does this need to be quoted? `<html lang="<%%= I18n.locale %>">`

Reply:
I'm inserting the value of `<%= I18n.locale %>`, which evaluates to the string "en" (or the symbol `:en`), so it doesn't need to be quoted here.

Reply:
@zzak I've updated the PR with some more context about what these changes are, what they do in the app, and some QA steps

Reply:
I'll update this

Reply:
Just to clarify, after updating this to `<html lang="<%%= I18n.locale %>">` the generated `application.html.erb` file will contain `<html lang="<% I18n.locale %>"`, whereas `<html lang=<%%= I18n.locale %>>` generates it as `<html lang=<% I18n.locale %>`. In both cases, when inspecting this in the browser it'll show as `<html lang="en">`.